content-src/asrouter/templates/Trailhead/_Trailhead.scss

```diff
@@ -80,13 +86,21 @@
     padding: 0;
 
     li {
-      background-position: left 4px;
+      background-position: left 6px;
```

Comment:
nit: let's just set these individually `background-position-x` `background-position-y` and only change the latter for the breakpoint.

Reply:
oh and handle RTL! 😉

Reply:
nvm, you already did 😄